core/environment.ts

```diff
@@ -118,3 +119,24 @@ export function getEnvironment(): Environment {
   }
   return currentEnvironment
 }
+
+const testRouter = atom<BaseRoute | undefined>()
+
+export function setBaseRoute(route: BaseRoute | undefined): void {
```

Comment:
`setBaseRoute` → `setBaseTestRoute`

Otherwise people may think that it is core router function.